### DISCUSSION TITLE
Absolute positioning

### DISCUSSION THREAD

**Discussion by AshDini** [rio-labs/rio#150] [General]
IS THERE ANY PLAN IN THE FURUE TO IMPLEMENT ABSOULUTE POSITIONING OF THE UI COMPONENTS ?

**Comment by mad-moo** [comment]
Please cut down on the caps :sweat_smile: 

What exactly are you looking for? You can already position elements wherever you want them by using the margin. If you want to position them relative to the screen, `rio.Overlay` can do that

**Comment by AshDini** [comment]
Sorry about the up case my keyboard has been playing up.
I will try the overlay function, see if it will help me to position the UI
widgets.

How can I direct the output to my Android mobile phone ?

Thanks for you prompt reply what is bír.

**Comment by mad-moo** [comment]
Run your app with `--public`, so for example `rio run --public`. This will allow other devices on the network to connect. You can then type your computer's IP & port into your phone's browser to see the app

**Comment by AshDini** [comment]
Thanks for the tip!
